### PR TITLE
Update Embedding Documentation

### PR DESCRIPTION
**Pull request by hSterz**
Improve Embedding Documentation (#643 ) by:
- Adding an example notebook
- Updating the embedding page in the documentation

### CODE REVIEW

File: docs/embeddings.md [hunk 1]
```diff
@@ -28,7 +27,12 @@ we want to delete. However, you cannot delete the default embedding.
 model.delete_embeddings('name')
 ```
 Please note, that if the active embedding is deleted the default embedding is set as the active embedding.
-
+## Training Embeddings
```
Comment (calpt) [inline docs/embeddings.md:30]:
```suggestion

## Training Embeddings
```

File: docs/embeddings.md [hunk 2]
```diff
@@ -28,7 +27,12 @@ we want to delete. However, you cannot delete the default embedding.
...
+Embeddings can only be trained with an adapter. To freeze all weights except for the embedding and the adapter:
+```python
+model.train_adapter('adapter_name', train_embeddings=True)
+```
+Except for the `train_embeddings` flag, the training is the same as for just training an adapter (see [Adapter Training](training.md)).
```
Comment (calpt) [inline docs/embeddings.md:35]:
```suggestion
Except for the `train_embeddings` flag, the training is the same as for just training an adapter (see [Adapter Training](training.md)).

```

File: notebooks/Adapter_With_Embeddings.ipynb [hunk 3]
```diff
@@ -0,0 +1,772 @@
+{
```
Comment (calpt) [inline notebooks/Adapter_With_Embeddings.ipynb:1]: Can we add a title and a short introduction explaining the goal of this notebook?

Comment (hSterz) [reply notebooks/Adapter_With_Embeddings.ipynb:1]: That is a very good point! I added title and introduction to the notebook

**Review by TimoImhof — APPROVED** [review]
Looks good to me! I fixed a few minor typos. With the requested changes incorporated, this is ready to go!

**Review by calpt — APPROVED** [review]

**Pull request merged**